test/parser/libqasm/test_libqasm.py

```diff
@@ -78,3 +109,56 @@ def test_error(parser: Parser) -> None:
 def test_wrong_gate_argument_number_or_types(parser: Parser, error_message: str, circuit_string: str) -> None:
     with pytest.raises(IOError, match=error_message):
         parser.circuit_from_string(circuit_string)
+
+
+@pytest.mark.parametrize(
+    ("expected_output", "circuit_string"),
+    [
+        (
+            """version 3.0
+
+qubit[1] q
+""",
+            """
+            version 3.0
+
+            qubit q
+
+            """,
+        ),
+        (
+            """version 3.0
+
+qubit[1] q
+bit[1] b
+""",
+            """
+            version 3.0
+
+            qubit q
+            bit b
+
+            """,
+        ),
+        (
+            """version 3.0
+
+bit[1] b
+""",
+            """
+            version 3.0
+
+            bit b
+
+            """,
```

Comment:
Same comments as above. Can we write all this in single lines? Also, do no use unnecessary blank lines at the end of input programs.